1-introduction/1.3.4-passwords-and-keys.md

```diff
@@ -43,7 +48,7 @@ SSH makes use of public key authentication to allow passwordless and non-interac
 
 ```bash
 # Generate SSH keys. Use the default file and empty passphrase for the keys.
-ssh-keygen
+ssh-keygen -b 4096
```

Comment:
adding this seems unnecessary as different situations might call for different keys and the command will prompt you anyway.

Reply:
I think we should actually change this to 

`ssh-keygen -t ed25519` 

the industry is moving away from SHA-1 keys. and I believe it is in OpenSSL 8.2 that you cannot even use SHA-1 keys without modifications. This also happens to be the version of OpenSSH that comes with most modern operating systems like ubu 22.04 and CentOS 9.

And as a bonus these keys are much smaller.